src/js/common/schemaform/FormPage.jsx

```diff
@@ -101,15 +103,16 @@ class FormPage extends React.Component {
     }
   }
   getEmptyState() {
-    return { formContext: { touched: {}, submitted: false, onEdit: this.props.onEdit, hideTitle: this.props.hideTitle } };
+    const formData = this.props.form[this.props.route.pageConfig.pageKey].data;
+    return { formContext: { touched: {}, submitted: false, onEdit: this.props.onEdit, hideTitle: this.props.hideTitle, formData } };
```

Comment:
Could get `onEdit` and `hideTitle` from destructuring `this.props` right before this to shorten this line. Also, `onEdit` doesn't seem to be among the `propTypes`.